schema/release-schema.json

```
@@ -371,6 +371,18 @@
         "eligibilityCriteria": {
           "title": "Eligibility criteria",
           "description": "A description of any eligibility criteria for potential suppliers.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "deprecated": {
+            "description": "This field is deprecated in favor of exclusionGrounds, which uses more standard terminology.",
+            "deprecatedVersion": "1.2"
+          }
+        },
+        "exclusionGrounds": {
+          "title": "Exclusion grounds",
+          "description": "The criteria regarding the situation of a tenderer that can lead to its exclusion from the process. Typical exclusion grounds include bankruptcy, presence on a blacklist or failure to pay taxes.",
```

In #901: 

> In https://www.procurementjourney.scot/route-3/develop-documents/exclusion-selection-and-award-criteria/exclusion-criteria, bankruptcy is in the "may ask" column, thus less typical than "criminal convictions". I'll update my proposal.

I agree that criminal convictions is a more familiar example.

```suggestion
          "description": "The criteria regarding the situation of a tenderer that can lead to its exclusion from the contracting process. For example: criminal convictions, bankruptcy, presence on a blacklist or failure to pay taxes.",
```